test/customHarvesterInstance.spec.js

```diff
@@ -0,0 +1,39 @@
+var request = require('supertest');
+var should = require('should');
+var harvester = require('../lib/harvester');
+
+var config = require('./config.js');
+var seeder = require('./seeder.js');
+
+/**
+ * This test case demonstrates how to setup test with custom harvester on different port
+ */
+describe('Custom harvester demo', function () {
+    var baseUrl = 'http://localhost:8001';
+    before(function () {
+        function customAppConfigurator(app) {
```

Comment:
not sure if this fn is necessary - it introduces a new pattern of pushing harvester.resource into a function for no real benefit. Because this is a demo, people will repeat this pattern and we'll see 2 ways of initializing harvester, which we don't want.